### PR TITLE
[W4.11e][T09-3]Ivan Ho Weng Kwong

### PR DESCRIPTION
**Pull request by ihwk1996**
UtilsTest.java: add test for isAnyNull(Object…​) method

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -43,4 +43,10 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNullTest() {
+        assertFalse(Utils.isAnyNull("a", "b", "c"));
+        assertTrue(Utils.isAnyNull(null, "b", "c"));
```
Comment (eugenepeh) [inline test/java/seedu/addressbook/common/UtilsTest.java:50]: Good! You've included both positive and negative test cases.

Can do with more test cases on other scenarios such as `empty` collection, other data type etc as well.

**Review by eugenepeh — COMMENTED** [review]
Good job on completing the activity.
Some comments added. Please close the PR after reading the comments.
